jdaviz/configs/rampviz/plugins/parsers.py

```diff
@@ -235,8 +243,24 @@ def _parse_hdulist(
         file_name = file_name or "Unknown HDU object"
 
     hdu = hdulist[1]  # extension containing the ramp
-    if hdu.header['NAXIS'] != 4:
-        raise ValueError(f"Expected a ramp with NAXIS=4 (with axes:"
+
+    if hdu.header['NAXIS'] == 2:
+        # this may be a calibrated Level 2 image:
```

Comment:
If this is a "may", do we need some sort of `if` statement here (rather than waiting until later to check for `SCI` ext)?

Reply:
The challenge I was having in writing this comment concisely and correctly is that a user could always try to load a 2D FITS image that isn't a Level 2 image. There are `if` statements within the method that gets called below this line, in the Imviz parser's method `_parse_image`. Do you have particular corner cases in mind that we should defend against?

Reply:
I don't have a particular corner case in mind, I just wanted to make sure something wasn't missed given the comment. Sounds like it's handled elsewhere.